.github/workflows/build.yml

```diff
@@ -74,63 +73,35 @@ jobs:
             build_type: Release
             code_coverage: false
             static: false
-    uses: ./.github/workflows/build_impl.yml
-    with:
-      runs_on: ${{ matrix.os }}
-      container: ${{ matrix.container }}
-      conan_profile: ${{ matrix.conan_profile }}
-      build_type: ${{ matrix.build_type }}
-      code_coverage: ${{ matrix.code_coverage }}
-      static: ${{ matrix.static }}
-      unit_tests: true
-      integration_tests: true
-      clio_server: true
-
-  test:
-    name: Run Tests
-    needs: build
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - os: heavy
-            conan_profile: gcc
-            build_type: Release
-            container:
-              image: rippleci/clio_ci:latest
-          - os: heavy
-            conan_profile: clang
-            build_type: Release
-            container:
-              image: rippleci/clio_ci:latest
-          - os: heavy
-            conan_profile: clang
-            build_type: Debug
-            container:
-              image: rippleci/clio_ci:latest
-          - os: macos15
-            conan_profile: apple_clang_16
-            build_type: Release
-    runs-on: ${{ matrix.os }}
-    container: ${{ matrix.container }}
-
     steps:
-      - name: Clean workdir
-        if: ${{ runner.os == 'macOS' }}
-        uses: kuznetsss/workspace-cleanup@80b9863b45562c148927c3d53621ef354e5ae7ce #v1.0
+      - uses: actions/checkout@v4
 
-      - uses: actions/download-artifact@v4
+      - name: Build
+        uses: ./.github/workflows/build_impl.yml
```

Comment:
Unfortunately, calling a reusable workflow doesn't work as a step of another workflow. I will think how to overcome this, maybe I will make our reusable workflows actions

Reply:
Added a new workflow which simply runs both of build and test